tests/integration/components/paper-form-test.js

```diff
@@ -50,12 +50,12 @@ test('form `onSubmit` action is invoked', function(assert) {
       {{form.input value=foo onChange=(action (mut foo)) label="Foo"}}
       {{form.input value=bar onChange=(action (mut bar)) label="Bar"}}
 
-      <button onclick={{action form.onSubmit}}>Submit</button>
+      <a {{action form.onSubmit}}>Submit</a>
```

Comment:
@tyleryasaka That's what happens by default, but I changed it to `<button type="button"`.